src/arm/overlays/BBAI-BB-BONE-4D4C-01-00A1.dts

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
I'll be looking for where the secondary copy of this is deleted and a common file is used. It would be easier for me to follow if that was the first step.

Reply:
OK, I think we are still stuck on getting this into one file? I think making it clear in the commit log where the hold-up is would go a long way in enabling someone else to pick it up.